e2e/test/scenarios/visualizations/pie_chart.cy.spec.js

```diff
@@ -46,3 +61,10 @@ function ensurePieChartRendered(rows, totalValue) {
     });
   });
 }
+
+function checkLegendItemOpacity(title, opacity) {
+  cy.findByTestId("chart-legend")
+    .findByTestId(`legend-item-${title}`)
+    .should("have.attr", "style")
+    .and("include", `opacity: ${opacity}`);
+}
```

Comment:
We generally don't write tests that assert on styles (they're generally too fragile). Is there a way we can re-write this to use roles instead?

Reply:
There wasn't really a role that made sense to use here, so instead I used the `aria-current` attribute, which seems very appropriate for this use case:

> When you have a group of related elements, such as several links in a breadcrumb or steps in a multi-step flow, with one element in the group styled differently from the others to indicate to the sighted user that this is the current element within its group, the aria-current should be used to inform the assistive technology user what has been indicated via styling ([MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current#description)).

Reply:
That looks good to me!